media/com_fabrik/js/tipsBootStrapMock.js

```diff
@@ -190,14 +218,23 @@ define(['jquery', 'fab/fabrik'], function (jQuery, Fabrik) {
                         .css({top: 0, left: 0, display: 'block'})
                         .appendTo(inside ? this.$element : document.body);
 
-                    pos = this.getPosition(inside);
-
-                    actualWidth = $tip[0].offsetWidth;
-                    actualHeight = $tip[0].offsetHeight;
-
-                    switch (inside ? placement.split(' ')[1] : placement) {
+                    var pos = this.getPosition(inside);
```

Comment:
I prefer this form of var definition i.e. on first use.

Reply:
These have all been changed to do that.